ci/conda/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
   build:
     - patch             [win]
     - python
-    - oce 0.17.*
+    - oce 0.17.2
```

Comment:
this is a wrong syntax. this will give you nasty errors ;) I was struggeling with it right now

Use ```oce ==0.17.2``` instead